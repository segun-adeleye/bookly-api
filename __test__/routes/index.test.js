'use strict';

it('should load index page', () =>
  request
    .get('/')
    .expect(200)
    .then(res => {
      expect(res.text).toMatchSnapshot();
    }));
