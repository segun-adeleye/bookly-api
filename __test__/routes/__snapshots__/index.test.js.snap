// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should load index page 1`] = `
"{
    \\"message\\": \\"Bookly. Let's know what you are reading.\\"
}"
`;
